storybookjs/react-native
# feat: nested grouping of stories with "group/story" syntax

Issue: #205 

## What I did

thanks to @Jonatthu for getting this started with #457

This adds an initial version of categories/nested stories in the sidebar

## How to test

- run example project and see the sidebar and storylist story

https://github.com/storybookjs/react-native/assets/3481514/b2c29fd8-972a-4667-8553-9de33dfaf0e8
